.buildkite/scripts/docs_push.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/ash
+# shellcheck shell=dash
+
+set -exo pipefail
+
+if [ "$BUILDKITE_BRANCH" != "master" ]; then
+  echo "nothing to do"
+  exit 0
+fi
+
+mkdir -p "$HOME/.ssh"
+ssh-keyscan github.com >> "$HOME/.ssh/known_hosts"
+git config --local user.email "buildkite@m3db.io"
+git config --local user.name "M3 Buildkite Bot"
+
+rm -rf site
```

Comment:
Can these lines be moved/call into the Makefile?

Reply:
So I looked into calling the makefile from the docs container and it turns out our Makefile makes a lot of assumptions about being run in `bash`, but alpine only has `ash`. I can convert the docs image to a debian one but then I'll have to make sure it has all the same mkdocs packages and the container will be heavier. Not sure if it's worth, wondering what you think.

Reply:
Yech. Isn’t worth it to do the work to update either right now. I’d suggest dropping TODOs here and by the makefile target reminding people to keep commands in sync and an explanation for how we might migrate to a common target if/when we have time